assets/scss/6-components/site-footer/site-footer.html

```diff
@@ -1,187 +1,92 @@
-          <a
-            href="https://mediakit.texastribune.org/"
-            title="Advertise"
-            class="advertise"
-            >Advertise</a
-          >
-        </li>
-        <li><a href="/">© 2019 The Texas Tribune</a></li>
+      <div class="has-b-btm-marg">
+        <span class="c-site-footer__logo c-icon has-text-yellow">
+          <svg aria-hidden="true" focusable="false">
+            <use xlink:href="#bug"></use>
+          </svg>
+        </span>
+      </div>
+
+      <div class="has-notch has-notch--thin has-bg-yellow has-b-btm-marg" aria-hidden="true"></div>
+      
+      <ul class="c-site-footer__links t-size-xs has-text-white has-text-hover-white">
+        <li class="has-xxxs-btm-marg"><a class="has-text-blue has-text-hover-blue" href="https://support.texastribune.org/donate">Donate</a></li>
+        <li class="has-xxxs-btm-marg"><a href="https://www.texastribune.org/contact/">Contact Us</a></li>
+        <li>&copy; 2019 The Texas Tribune</li>
       </ul>
     </div>
-    <div id="footer-sections" class="col hide_until--s">
-      <h5 class="c-site-footer__header">Topics</h5>
-      <ul>
-        <li>
-          <a href="/topics/congress/" data-section="congress">Congress</a>
-        </li>
-        <li>
-          <a href="/topics/courts/" data-section="courts">Courts</a>
-        </li>
-        <li>
-          <a href="/topics/criminal-justice/" data-section="criminal-justice"
-            >Criminal justice</a
-          >
-        </li>
-        <li>
-          <a href="/topics/demographics/" data-section="demographics"
-            >Demographics</a
-          >
-        </li>
-        <li>
-          <a href="/topics/economy/" data-section="economy">Economy</a>
-        </li>
-        <li>
-          <a href="/topics/energy/" data-section="energy">Energy</a>
-        </li>
-        <li>
-          <a href="/topics/environment/" data-section="environment"
-            >Environment</a
-          >
-        </li>
-        <li>
-          <a href="/topics/health-care/" data-section="health-care"
-            >Health care</a
-          >
-        </li>
-        <li>
-          <a href="/topics/higher-education/" data-section="higher-education"
-            >Higher education</a
-          >
-        </li>
-        <li>
-          <a href="/topics/immigration/" data-section="immigration"
-            >Immigration</a
-          >
-        </li>
-        <li>
-          <a href="/topics/politics/" data-section="politics">Politics</a>
-        </li>
-        <li>
-          <a href="/topics/public-education/" data-section="public-education"
-            >Public education</a
-          >
-        </li>
-        <li>
-          <a href="/topics/state-government/" data-section="state-government"
-            >State government</a
-          >
-        </li>
+    
+    <div class="col t-size-xs">
+      <h5 class="c-site-footer__header has-text-yellow t-uppercase has-xs-btm-marg"><strong>Heading</strong></h5>
```

Comment:
You probably saw, but ugh stupid bulma is kicking in for that h5 tag font-weight. We _could_ put a note that `<strong/>` doesn't really need to be there.

Reply:
Done.